config/application.rb

```diff
@@ -33,5 +33,9 @@ class Application < Rails::Application
     config.lograge.enabled = true
 
     config.secret_path = Rails.root.join('run/secrets')
+    
+    Raven.configure do |config|
```

Comment:
Can you elaborate on why this is being added?

Reply:
I added this because I got an error `sentry-raven DSN not set`. This was the solution I found on the internet to get the server to run locally. I plan on dropping it when I finish the PR.